web/src/hotkey.js

```diff
@@ -582,6 +583,11 @@ export function process_shift_tab_key() {
         return emoji_picker.navigate("shift_tab");
     }
 
+    if ($("#stream_message_recipient_topic").is(":focus")) {
+        compose_recipient.open_compose_stream_dropup();
+        return true;
+    }
```

Comment:
This feels great! We may want to do a similar check in the "Move topic" modal, since we start focus in the topic input there too, but that can be a follow-up.